spec/models/conference_spec.rb

```diff
@@ -1656,4 +1656,23 @@
       expect{ conference.save }.to change{ EventSchedule.count }.from(2).to(1)
     end
   end
+
+  describe '#upcoming_conferences', blah: true do
```

Comment:
Since this is a scope, it's a class method, and it should be described as `.upcoming` (with a dot, not with a `#`)